core/primitives/bitvec/bitvec.hpp

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_CORE_PRIMITIVES_RLE_BITSET_RLE_BITVEC_HPP
+#define CPP_FILECOIN_CORE_PRIMITIVES_RLE_BITSET_RLE_BITVEC_HPP
+
+#include <cstdint>
+#include <vector>
+
+namespace fc::primitives {
```

Comment:
Namespace and directory doesn't match, it is not required but recommended. Please consider putting `BitvecReader` class in `fc::primitives::bitvec`.

Reply:
Fixed